src/sdk.ts

```diff
@@ -1,17 +1,17 @@
 export type GetEventMetadataOptions = { eventName: string };
 export type GetEventMetadataResult = {
-  firstSeenAt: string;
-  lastSeenAt: string;
+  received: boolean;
```

Comment:
nit: missing comment here to describe what this is and how should be used

Reply:
Added 👍